src/VirtualClient/VirtualClient.Main/profiles/MONITORS-GPU-AMD-SMI.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
There is https://github.com/microsoft/VirtualClient/blob/main/src/VirtualClient/VirtualClient.Main/profiles/MONITORS-GPU-AMD.json
That you should be able to fit in.

Reply:
This profile has LspciMonitor which does not run on Linux. Hence created a separate profile for AMD SMI which can now support both Windows and Linux.

Reply:
Sorry for the late reply, the lspci was designed for linux, we added windows binaries just for compatibility. Please let me know what you see in your linux system. If it's broken we should fix it

Reply:
I have now combined both into 1 profile. Please check the latest commit for review.